yaml/pois.yaml

```diff
@@ -835,7 +835,14 @@ filters:
   ############################################################
   # TIER 6
   ############################################################
-  # allotments - no POI
+  # allotments
+  - filter:
+      landuse: allotments
+    min_zoom: 16
```

Comment:
Gardens and allotments are basically the same, they should share the same zoom:

>     min_zoom: { min: [ { max: [ 12, { col: zoom }, *tier6_min_zoom ] }, 16 ] }

Or better yet just expand the existing garden filter, like I'll comment?